src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -140,6 +140,30 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="ImportOrder"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * import java.io.IOException; // OK
+ * import java.net.URL; // OK
+ *
+ * import java.io.IOException; // violation, extra separation before 'java.io.IOException'.
```

Comment:
`'java.io.IOException'` no need to add this, it is pretty clear what import this comment is referring to

Reply:
done